src/gui/static/src/app/app.loadWallet.ts

```diff
@@ -705,19 +710,46 @@ export class LoadWalletComponent implements OnInit {
         .map((res:Response) => res.json())
         .subscribe(
             response => {
-                //Hide load wallet seed popup
-                this.loadSeedIsVisible = false;
-                //Load wallet for refresh list
-                this.loadWallet();
-            },
-            err => {
-                if(err._body.indexOf("duplicate wallet ") !=-1){
```

Comment:
The duplicate wallet checking is gone, will stuck when try to load wallet from same seed.